app/tasks/kis.py

```
@@ -0,0 +1,737 @@
+
+    return asyncio.run(_run())
+
+
+@shared_task(name="kis.analyze_domestic_stock_task", bind=True)
+def analyze_domestic_stock_task(self, symbol: str) -> dict:
+    """단일 국내 주식 분석 실행"""
+    return asyncio.run(_analyze_domestic_stock_async(symbol))
+
+
+@shared_task(name="kis.execute_domestic_buy_order_task", bind=True)
+def execute_domestic_buy_order_task(self, symbol: str) -> dict:
```

Ruff (0.14.6)

Unused function argument: `self`

(ARG001)